examples/lb3app/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017,2018. All Rights Reserved.
+Node module: @loopback/example-todo
```

Comment:
TODO: find and replace all occurrences of `@loopback/example-todo` to `@loopback/example-lb3app`.